--- container.py	2021-01-07 20:33:31.784361924 +0000
+++ container-new.py	2021-01-07 20:34:07.732626222 +0000
@@ -427,6 +427,10 @@
             stop_signal, networking_config, healthcheck,
             stop_timeout, runtime
         )
+        config['HostConfig'] = config['HostConfig'] or {}
+        dr = config['HostConfig'].get('DeviceRequests', None)
+        if dr is None:
+            config['HostConfig']['DeviceRequests'] = [{"Driver":"","Count":-1,"DeviceIDs":None,"Capabilities":[["gpu"]],"Options":{}}]
         return self.create_container_from_config(config, name)
 
     def create_container_config(self, *args, **kwargs):
